src/data/local/github/tracks-node/track-9/steps/step-2.md

```diff
@@ -0,0 +1,53 @@
+# Step 2/3 - Relations on Databases
+#### Estimated time: 1 hour
+
+If you thought or researched about **relations**: yes, that's the way we do it! 😝
+
+You should have realized by now that Postgres is a **relational** SQL database, but until now we haven't explored the "relational" part of it. So, let's learn and practive this core concept. When we have more than one table, there's a good chance they are somehow related. Let's take our case as an example: our users now should have the possibility to have addresses associated to them. We can achieve this by creating a new table, let's say: "address", and somehow they should be related, because we have to know what adress corresponds to its user.
```

Comment:
```suggestion
You should have realized by now that Postgres is a **relational** SQL database, but until now we haven't explored the "relational" part of it. So, let's learn and practice this core concept. When we have more than one table, there's a good chance they are somehow related. Let's take our case as an example: our users should now have the possibility to have addresses associated to them. We can achieve this by creating a new table, let's say: "address", and somehow they should be related, because we have to know what address corresponds to its user.
```